doc/modules/linear_model.rst

```diff
@@ -734,12 +734,12 @@ regularization.
 As an optimization problem, binary class L2 penalized logistic regression
 minimizes the following cost function:
 
-.. math:: \underset{w, c}{min\,} \frac{1}{2}w^T w + C \sum_{i=1}^n \log(\exp(- y_i (X_i^T w + c)) + 1) .
+.. math:: \underset{w, c}{\min\,} \frac{1}{2}w^T w + C \sum_{i=1}^n \log(\exp(- y_i (X_i^T w + c)) + 1) .
```

Comment:
Is the `\underset` still necessary? Wouldn't `\min_{w, c}\,` work? I'm not sure if we are in inline or display latex mode there (see https://tex.stackexchange.com/a/49742/73431)

Reply:
Hi,
I will work some more on this linear_model.rst file: indeed they are other issues, including the one you mention.
I'll be off the coming two weeks, but I think I can do that in the second half of July (or may be before, though less likely).
Best,
Joseph